extensions/renderer/resources/app_window_custom_bindings.js

```diff
@@ -31,7 +31,7 @@ var try_hidden = function (view) {
 
 var try_nw = function (view) {
   if (view.nw)
-    return view;
+    view.nw.Window.get(); //construct the window object for NWJS#5294
   return privates(view);
```

Comment:
This would have side effects in other places calling try_nw(). It should be handled in getAll().